doc/contribute/bin_blobs.rst

```diff
@@ -42,22 +36,22 @@ Blobs must be hosted on the Internet and managed by third-party infrastructure.
 Two potential examples are Git repositories and web servers managed by
 individual hardware vendors.
 
-The Zephyr Project will not host binary blobs in its Git repositories or
+The Zephyr Project does not host binary blobs in its Git repositories or
 anywhere else.
 
 Fetching blobs
 ==============
 
-Blobs will be fetched from official third-party sources via a west extension
-command which users will be able to run to this effect. The extension command
-implementation will be hosted in the main zephyr repository. This is required to
+Blobs are fetched from official third-party sources via a :ref:`west extension
+command <west-blobs>`, which users can run to this effect. The extension command
+implementation is hosted in the main zephyr repository.  This is required to
 give the project overall control over the mechanism used to fetch the blobs.
 
-The blobs themselves must be specified in the :file:`module.yml` files included
-in separate Zephyr :ref:`module repositories <modules>` maintained by their
-respective vendors.  This means that in order to submit a binary blob to the
-upstream Zephyr distribution, a module repository must exist first or be created
-as part of the submission process.
+The blobs themselves must be specified in the :ref:`module.yml
+<modules-bin-blobs>` files included in separate Zephyr :ref:`module
+repositories <modules>` maintained by their respective vendors.  This means that
+in order to submit a binary blob to the upstream Zephyr distribution, a module
```

Comment:
This may cause confusion, as it could give the impression that the binary blob itself is submitted, but we just stated above that Zephyr project doesn't host binary blobs.

Maybe make it clearer that it's a reference to the binary blob that is submitted.